lib/run-script.js

```diff
@@ -125,9 +125,16 @@ function run (pkg, wd, cmd, args, cb) {
   } else {
     if (!pkg.scripts[cmd]) {
       if (cmd === "test") {
-        pkg.scripts.test = "echo \"Error: no test specified\"";
+        pkg.scripts.test = "echo \"Error: no test specified\""
+      } else if (cmd === "env") {
+        log.verbose("run-script using default multiplatform env")
```

Comment:
The problem is that `env` and `SET` are actually semantically quite distinct -- you can pass arguments to `env` and they'll be treated as a command invocation in that environment (which is why `env` is so useful on the shebang line in scripts). `SET` doesn't have this property, and it would be nice to be able to tell from the logs which you've gotten.